harper-core/src/linting/phrase_set_corrections/mod.rs

```diff
@@ -279,6 +279,42 @@ pub fn lint_group() -> LintGroup {
             "Corrects the eggcorn `piggy bag` to `piggyback`, which is the proper term for riding on someone’s back or using an existing system.",
             LintKind::Eggcorn
         ),
+
+        // General litotes (double negatives) → direct positive
+        "LitotesDirectPositive" => (
+            &[
+                ("not uncommon", "common"),
+                ("not unusual", "common"),
+                ("not insignificant", "significant"),
+                ("not unimportant", "important"),
+                ("not unlikely", "likely"),
+                ("not infrequent", "frequent"),
+                ("not inaccurate", "accurate"),
+                ("not unclear", "clear"),
+                ("not irrelevant", "relevant"),
+                ("not unpredictable", "predictable"),
+                ("not inadequate", "adequate"),
+                ("not unpleasant", "pleasant"),
+                ("not unreasonable", "reasonable"),
+                ("not impossible", "possible"),
+            ],
+            "Prefer a direct form over litotes like `not uncommon`.",
+            "Rewrites common double negatives (litotes) to a clearer direct positive.",
+            LintKind::WordChoice
+        ),
+
+        // Redundant degree modifiers on positives (double positives) → base form
+        "RedundantSuperlatives" => (
+            &[
+                ("more optimal", "optimal"),
+                ("most optimal", "optimal"),
+                ("more ideal", "ideal"),
+                ("most ideal", "ideal"),
```

Comment:
I think I or maybe somebody else made a lint for "more preferable". Yes I can see it now (-: Maybe these should be rolled in together. 

<img width="536" height="128" alt="Image" src="https://github.com/user-attachments/assets/b8b90d0c-1eee-4018-ba63-334e7d735e69" />